repo/fsrepo/fsrepo.go

```diff
@@ -460,9 +457,9 @@ func (r *FSRepo) Config() *config.Config {
 	defer packageLock.Unlock()
 
 	if r.closed {
-		panic("repo is closed")
```

Comment:
the other ops return an error. i'm don't this should be panic. all sort of code all over the place calls `repo.Config()`